src/DataGrid.tsx

```diff
@@ -1138,6 +1166,13 @@ function DataGrid<R, SR, K extends Key>(
     return rowElements;
   }
 
+  if (gridWidth !== prevGridWidth) {
+    setPrevGridWidth(gridWidth);
+    if (columns.length === viewportColumns.length) {
+      setMeasuredColumnWidths(new Map());
```

Comment:
1. grid width changes (1st render)
2. we update the previous grid width state + reset measured col widths (2nd render)
3. we measure col widths and update the state (3rd render)

Can we perhaps combine 2. and 3.?